File: Java/bee 1071.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int a, b;
        a = sc.nextInt();
        b = sc.nextInt();

        if(b < a){
            int aux = a;
            a = b;
            b = aux;
        }

        int soma = 0;
        for(int i = a+1; i < b; i++){
            if(i % 2 != 0){
                soma += i;
            }
        }
        System.out.println(soma);
    }
}

File: C++/bee 1071.cpp
#include <iostream>
#include <iomanip>
#include <stdio.h>

using namespace std;

int main()
{
    int x, y;
    int soma = 0;
    cin >> x >> y;
    if(x > y){
        int aux = x;
        x = y;
        y = aux;
    }
    for(int i = x+1; i < y; i++){
        if(i % 2 != 0){
            soma += i;
        }
    }
    cout << soma << endl;
}
